app/AccountancyModule/UnitAccountModule/presenters/BudgetPresenter.php

```diff
@@ -24,14 +24,24 @@ public function __construct(BudgetService $budgetService)
     protected function beforeRender() : void
     {
         parent::beforeRender();
-        $this->setLayout('layout.new');
+        $this->setLayout('layout2');
     }
 
     public function renderDefault(?int $year = null) : void
     {
         $this->template->setParameters([
             'categories' => $this->budgetService->getCategories($this->unitId->toInt()),
             'unitPairs'  => $this->unitService->getReadUnits($this->user),
+            'year'       => $year,
+        ]);
+    }
+
+    public function renderAdd(?int $year = null) : void
+    {
+        /** @var BaseForm $form */
+        $form = $this['addCategoryForm'];
+        $form->setDefaultValues([
```

Comment:
@fmasa jak nejlépe tady nastavit výchozí hodnotu? tohle mi nechce uznat wercker

Reply:
IMO to PHPStan hlásí správně. Metoda `Form::setDefaultValues()` neexistuje, pravděpodobně hledáš `Form::setDefaults()`

Reply:
ja jsem to postupne nekoolikrat menil. uz ok ale neprochazi mi ty akceptacni testy :/